File: Administrativo/spring-erp-servicio/net/royal/spring/programasocial/dominio/filtros/FiltroConsumo.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.rrhh.dominio.dto
{
    public class FiltroConsumo
    {
       public String codigoInstitucion { get; set; }
       public String nombreInstitucion{ get; set; }

        public String codigoPeriodo { get; set; }

        public Nullable<DateTime> fechainicial { get; set; }

        public Nullable<DateTime> fechafinal { get; set; }

        public String estado { get; set; }




        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroConsumo()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/programasocial/consumo/dominio/FiltroConsumo.ts
import { ParametroPaginacionGenerico } from '../../../shared/dominio/ParametroPaginacionGenerico';

export class FiltroConsumo {

    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    codigoInstitucion: string;
    nombreInstitucion: string;
    codigoPeriodo: string;
    fechainicial: Date;
    fechafinal: Date;
    estado: string;

}
